src/components/drawer/ComponentDetailPanel.vue

```diff
@@ -193,6 +193,21 @@ async function save() {
   });
 }
 
+/**
+ * Get real type from definition.
```

Comment:
Get type of component value. Can be possibly different from component type given in definition.

Reply:
done